issue_processing/jira_issue_processing.py

```diff
@@ -178,30 +178,56 @@ def parse_xml(issue_data, persons):
                 d = datetime.strptime(resolveDate, '%a, %d %b %Y %H:%M:%S +0000')
                 issue['resolveDate'] = d.strftime('%Y-%m-%d %H:%M:%S')
 
+            title = issue_x.getElementsByTagName('title')[0]
+            issue['title'] = title.firstChild.data
+
             link = issue_x.getElementsByTagName('link')[0]
             issue['url'] = link.firstChild.data
 
             type = issue_x.getElementsByTagName('type')[0]
             issue['type'] = type.firstChild.data
+            # new consistent format with GitHub issues. Not supported by the network library yet
```

Comment:
maybe also add a `TODO` here as the `new` parts should replace the current ones later, I guess. (Or I am wrong here?)

This applies also for the following comments below.